packages/api2/src/auth/strategies/eth-signature.strategy.ts

```diff
@@ -30,11 +30,8 @@ export class EthSignatureStrategy extends PassportStrategy(
    * @returns
    */
   async validate(identityEthAddress: string, signature: string): Promise<any> {
-    let user;
-    // Check if user exists
-    try {
-      user = await this.usersService.findOneByEth(identityEthAddress);
-    } catch (e) {
```

Comment:
@kristoferlund This code has logical problem and it had caused failing some unit tests